### PR TITLE
Drop private imports on `test_utils.py`

### PR DESCRIPTION
**Pull request by T-256**
# Summary

closes https://github.com/encode/httpx/issues/2492

`get_ca_bundle_from_env` moved to where it's calling.

# Checklist

- [x] I understand that this PR may be closed in case there was no previous discussion. (This doesn't apply to typos!)
- [x] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
- [ ] I've updated the documentation accordingly.

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-    URLPattern,
-    get_ca_bundle_from_env,
-    get_environment_proxies,
-    guess_json_utf,
-    is_https_redirect,
-    obfuscate_sensitive_headers,
-    parse_header_links,
-    same_origin,
-)
+from httpx._utils import guess_json_utf  # has not any trace in public api
```
Comment (T-256) [inline tests/test_utils.py:11]: It has not any trace in public api.

File: tests/test_utils.py [hunk 2]
```diff
@@ -3,18 +3,13 @@
...
-    URLPattern,
-    get_ca_bundle_from_env,
-    get_environment_proxies,
-    guess_json_utf,
-    is_https_redirect,
-    obfuscate_sensitive_headers,
-    parse_header_links,
-    same_origin,
+    get_ca_bundle_from_env,  # catching `ssl.SSLError` is insufficient.
+    guess_json_utf  # has not any trace in public api.
```
Comment (T-256) [inline tests/test_utils.py:12]:
```suggestion
    guess_json_utf,  # has not any trace in public api.
```

**Comment by T-256** [comment]
Done

**Comment by karpetrosyan** [comment]
Please optimize the number of git commits. 
You can even do fewer commits or force push them at the end so that we only see a few commits rather than the commit army.
I believe it will make the pull request more attractive and easier to review.

**Comment by T-256** [comment]
> but switch to using private _accessors_.

previous PR like [this](https://github.com/encode/httpx/pull/2850/files#diff-d0fbc1167934498ad6894c6a6c6281f3bb91e979cf12502cf0c93090219acc98R37) also used private accessor, so I thought it's no problem using them in tests at all.

**Comment by T-256** [comment]
close in favor of separating to chunks...

**Pull request closed without merging**